runner.go

```diff
@@ -83,7 +83,7 @@ func Run(tasks []*Task, options ...RunOption) {
 	}
 
 	if len(tasks) == 0 {
-		panic(fmt.Errorf("no tasks specified"))
+		log.Fatalln("No tasks specified")
```

Comment:
nit: consistent lowercasing?

Reply:
I'm for consistently upcasing?

Reply:
i wish we didn't change it. either way, these aren't consistent now